### PR TITLE
Examples: Make domElement to a mandatory parameter for all controls.

### PR DESCRIPTION
**Pull request by Mugen87**
Replacement of #16336

This PR makes `domElement` to a mandatory ctor parameter. No default value (like `document`) is set anymore. `OrbitControls` and `TrackballControls` have an additional check which warns users when they use `document` as a ctor argument. The reason for this is explained here: https://github.com/mrdoob/three.js/pull/16326#issue-273211319

### CODE REVIEW

**Comment by mrdoob** [comment]
Looking great!

File: editor/js/EditorControls.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 THREE.EditorControls = function ( object, domElement ) {
 
-	domElement = ( domElement !== undefined ) ? domElement : document;
+	domElement = domElement;
```
Comment (mrdoob) [inline editor/js/EditorControls.js]: 😀

File: examples/js/controls/TransformControls.js [hunk 2]
```diff
@@ -4,9 +4,16 @@
...
+
+		console.warn( 'THREE.TransformControls: The second parameter "domElement" is now mandatory.' );
+		domElement = document;
+
+	}
+
 	THREE.Object3D.call( this );
 
-	domElement = ( domElement !== undefined ) ? domElement : document;
+	domElement = domElement;
```
Comment (mrdoob) [inline examples/js/controls/TransformControls.js]: 😀

File: examples/jsm/controls/TransformControls.js [hunk 3]
```diff
@@ -27,9 +27,16 @@ import {
...
+
+		console.warn( 'THREE.TransformControls: The second parameter "domElement" is now mandatory.' );
+		domElement = document;
+
+	}
+
 	Object3D.call( this );
 
-	domElement = ( domElement !== undefined ) ? domElement : document;
+	domElement = domElement;
```
Comment (mrdoob) [inline examples/jsm/controls/TransformControls.js]: 😀

**Comment by mrdoob** [comment]
Thanks!

**Comment by SalazarRiva** [comment]
I'm sorry, I'm a bit lost with this change here. I get it is supposed to get something fixed, but I do not get how I'm supposed to handle this - if up till now I could get away with controls = new THREE.OrbitControls( camera ) - where should I add a change.  Is there an example somewhere with what changes I should do?

**Comment by Mugen87** [comment]
Instead of 
```js
THREE.OrbitControls( camera );
```
do this:
```js
THREE.OrbitControls( camera, renderer.domElement );
```

**Pull request merged**